avocado/utils/iso9660.py

```diff
@@ -311,8 +312,11 @@ def __init__(self, path):
         """
         super(Iso9660Mount, self).__init__(path)
         self._mnt_dir = tempfile.mkdtemp(prefix='avocado_' + __name__)
-        process.run('mount -t iso9660 -v -o loop,ro %s %s' %
-                    (path, self.mnt_dir), sudo=True)
+        fs_type = 'iso9660'
```

Comment:
This is a matter of preference. I like this construct better than using if and else, which accomplishes basically the same thing. You'll see that in other parts of avocado. If I must, I can do it.